include/osdetection

```diff
@@ -273,6 +273,12 @@
                             OS_VERSION_FULL=$(grep "^VERSION=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
                             OS_NAME="Ubuntu"
                         ;;
+                        "rosa")
+                            LINUX_VERSION="ROSA Linux"
+                            OS_VERSION=$(grep "^VERSION_ID=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
+                            OS_VERSION_FULL=$(grep "^VERSION=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
+                            OS_NAME="ROSA Desktop Fresh R11.1"
```

Comment:
This should probably just be "ROSA Linux" since I'm guessing "ROSA Desktop Fresh R11.1" is the full OS version?

The list is also organized A-Z.

Reply:
Thanks @konstruktoid - I agree; Resolved conflict and applied changes.